src/bevy_ui.rs

```diff
@@ -32,13 +33,22 @@ pub fn compute_blur_regions<const N: usize>(
             continue;
         };
 
-        for (node, transform) in &nodes {
+        let viewport_size = window.size() / ui_scale.0;
+
+        for (node, transform, border_radius) in &nodes {
             let translation = transform.translation();
             let region = Rect::from_center_size(
                 translation.xy() * window.scale_factor(),
                 node.size() * window.scale_factor(),
             );
-            blur_regions.blur(region);
+            let resolved = [
+                border_radius.top_left,
+                border_radius.top_right,
+                border_radius.bottom_right,
+                border_radius.bottom_left,
+            ]
+            .map(|v| v.resolve(node.size().y, viewport_size).unwrap_or(0.0));
```

Comment:
```suggestion
            .map(|v| v.resolve(node.size().y, viewport_size).unwrap_or(0.0) * window.scale_factor());
```

Resolving the Val returns logical pixels, but the shader expects physical pixels.

Without converting to physical the blur comes outside of the radius, check out the bottom right corner in this screenshot:
![Screenshot 2024-09-17 at 20 51 09](https://github.com/user-attachments/assets/b9d8e8d4-cb5a-4ae8-b786-8819df5bcdd1)